frontend/src/components/news-page/News.tsx

```diff
@@ -1,35 +1,46 @@
 import React, { useState, useEffect } from "react";
 import { getTopWhaleVideos, RedditVideo } from "../../clients/redditApiClient";
+import { NewsCard } from "./NewsCard";
+import "./News.scss";
 
 export const News: React.FunctionComponent = () => {
   const [articlesList, setArticlesList] = useState<RedditVideo[]>();
```

Comment:
I know you didn't name this one, but could we rename it to just `videos`? The "list" aspect is implied by the plural, and they represent full video posts:
```suggestion
  const [videos, setVideos] = useState<RedditVideo[]>();
```